addons/website/views/website_templates.xml

```diff
@@ -546,13 +564,15 @@
                             <!-- Navbar -->
                             <li>
                                 <t t-call="website.navbar_nav">
-                                    <t t-set="_nav_class" t-valuef="my-3"/>
+                                    <t t-set="mobile_or_offcanvas" t-valuef="True"/>
+                                    <t t-set="header_offcanvas" t-valuef="True"/>
```

Comment:
- I'm not sure to understand what `mobile_or_offcanvas` represents 🤔
In both cases the megamenu would be in an offcanvas menu, right ?
So why not call it `is_in_offcanvas` ?

- This variable only triggers the left position of the offcanvas, right ? If so, perhaps it would be better to call it `offcanvas_is_leftside` or something like this ? 🤔

```suggestion
                                    <t t-set="is_in_offcanvas" t-valuef="True"/>
                                    <t t-set="offcanvas_is_leftside" t-valuef="True"/>
```

Reply:
Much better indeed, thanks